app/controllers/thread_controller.php

```diff
@@ -36,35 +35,41 @@ public function create()
     public function index()
     {
         $page = Param::get('page', 1);
-
         $pagination = new SimplePagination($page, self::MAX_ITEMS_PER_PAGE);
 
-        $threads = Thread::getAll($pagination->start_index -1, $pagination->count + 1);
+        $threads = Thread::getAll($pagination->start_index -1, 
+                                    $pagination->count + 1);
+        
         $pagination->checkLastPage($threads);
-       
         $total = Thread::countAll();
         $num_pages = ceil($total / self::MAX_ITEMS_PER_PAGE);
-
         $this->set(get_defined_vars()); 
     }
 
-     public function view()
+    public function view()
     {
         $page = Param::get('page', 1);
-
         $pagination = new SimplePagination($page, self::MAX_ITEMS_PER_PAGE) ;
 
-        $thread = Thread::get(Param::get('thread_id'));
-        $comments = $thread->getComments($pagination->start_index -1, $pagination->count + 1);
+        $thread_id = Param::get('thread_id');
+        $_SESSION['thread_id'] = $thread_id;
```

Comment:
You could apply the setter that I mentioned <a href='https://github.com/adrianneperedo/board/pull/2/files#r26107169'>here</a> po